contract/test/Tips.ts

```diff
@@ -0,0 +1,108 @@
+import { ethers } from "hardhat";
+import { Tips } from "../typechain-types";
+import { expect } from "chai"
+import { SignerWithAddress } from "@nomicfoundation/hardhat-ethers/signers";
+
+describe("Tips", function () {
+  let tips: Tips
+  let companyOwner: SignerWithAddress, member1: SignerWithAddress, member2: SignerWithAddress;
+  let recipientAddress: Array<string>;
+  const TIP_AMOUNT = ethers.parseEther("20");
+
+  beforeEach(async function () {
+    // Get signers for testing accounts
+    [companyOwner, member1, member2] = await ethers.getSigners();
```

Comment:
Why you have `companyOwner `it's not evident that only the company owner can send or push tips. 
You can rename it into `sender`